testing/sdk_dcos.py

```diff
@@ -0,0 +1,28 @@
+'''Utilities relating to getting information about DC/OS itself
+
+************************************************************************
+FOR THE TIME BEING WHATEVER MODIFICATIONS ARE APPLIED TO THIS FILE
+SHOULD ALSO BE APPLIED TO sdk_dcos IN ANY OTHER PARTNER REPOS
+************************************************************************
+'''
+from enum import Enum
+
+import sdk_cmd
+
+
+class DCOS_SECURITY(Enum):
+    disabled = 1
+    permissive = 2
+    strict = 3
+
+
+def get_metadata():
+    return sdk_cmd.cluster_request('GET',
+                                   'dcos-metadata/bootstrap-config.json',
```

Comment:
we already call another endpoint under `dcos-metadata` in `sdk_utils`, may be just move this code in to `sdk_utils.py`? if you think `sdk_dcos` may grow in future, then keep this 👍